docs/api/cookies.md

```diff
@@ -55,7 +64,7 @@ expired.
 
 The following methods are available on instances of `Cookies`:
 
-#### `cookies.get(filter, callback)`
```

Comment:
For now, we don't want to remove callback information from the docs; we want to have both side by side and just add 
> `**[Deprecated Soon](promisification.md)**`

at the bottom of the callback versions

Reply:
All four restored w/deprecation message in 1a452e175562b143cfd555ec043d3a50c60cb29f